frontend/components/idioms.jsx

```diff
@@ -7,41 +7,41 @@
   Some references used:
   https://claritydev.net/blog/the-most-common-mistakes-when-using-react
 */
-import { useEffect } from 'react'
+import { useEffect } from 'react';
+
+export const ShowButton = ({ buttonText }) => {
+  <button>{buttonText}</button>;
+};
 
 export function FunctionsAsComponents({ buttonText = 'Start Now' }) {
-  const showButton = () => {
-    <button>
-      {buttonText}
-    </button>
-  }
-
-  return (<div>
-    {showButton()}
-  </div>)
+  return (
+    <div>
+      <ShowButton buttonText={buttonText} />
+    </div>
+  );
 }
 
 export function objectCopying() {
-  const object = { a: { c: 1 }, b: 2 }
-  const copy = { ...object }
-  copy.a.c = 2
-  return { ...object }
+  const object = { a: { c: 1 }, b: 2 };
+  const copy = object; // doesn't make sense to deconstruct for nothing
+  copy['a']['c'] = 2; // not sure if this is better though feels the same
+  return object;
```

Comment:
This is not returning a copy, this is the same object. This is an incorrect answer.